source/core/slang-gcc-compiler-util.cpp

```diff
@@ -369,7 +369,8 @@ static SlangResult _parseGCCFamilyLine(const UnownedStringSlice& line, LineParse
 
     if (options.sourceType == SourceType::CPP)
     {
-        cmdLine.addArg("-std=c++11");
+        // Need C++14 for partial specialization
+        cmdLine.addArg("-std=c++14");
```

Comment:
Ideally our C++ output should be fairly conservative in what it required. I'd think that C++11 would be a fairly high version to require, and would prefer that we make things work with C++98 if at all possible.

Obviously if we aspire to have C output work some day, then we'd need to reduce our reliance on fancy C++ features to near-zero anyway.

That can be addressed in the long run.